### PR TITLE
feat: navigation bar

### PR DESCRIPTION
**Pull request by zohaibwebdev**

### CODE REVIEW

**Review by awaisshahid965 — CHANGES_REQUESTED** [review]
Need to fix

**Review by awaisshahid965 — CHANGES_REQUESTED** [review]
Push changes then request review

File: shared/types/icon.type.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface IIconType {
+    height: number
```
Comment (awaisshahid965) [inline shared/types/icon.type.ts]: Change file names to .types.ts

Comment (zohaibwebdev) [reply shared/types/icon.type.ts]: ok

File: shared/icons/logo.icon.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { IIconType } from '../types/icon.type'
+
+const LogoIcon: React.FC<IIconType> = ({ width = '90', height = '30' }) => {
```
Comment (awaisshahid965) [inline shared/icons/logo.icon.tsx]: if interface members are number, why their default values are in string?

File: shared/types/icon.type.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export interface IIconType {
```
Comment (awaisshahid965) [inline shared/types/icon.type.ts]: I think 'I' is added for interface, and we add **type** for a type. I don't think this is behaving as both interface and type so choose name accordingly like **IIconProps** if iterrface or **IconProps** if type.

File: shared/icons/logo.icon.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    return (
+        <svg
+            width={+width}
```
Comment (awaisshahid965) [inline shared/icons/logo.icon.tsx]: This is not okay as width is already a number

**Review by awaisshahid965 — APPROVED** [review]

**Review by awaisshahid965 — APPROVED** [review]

**Pull request merged**